tez-dag/src/main/java/org/apache/tez/dag/app/dag/event/DAGEventCounterUpdate.java

```diff
@@ -56,5 +56,9 @@ public Enum<?> getCounterKey() {
     public long getIncrementValue() {
       return incrValue;
     }
+
+    public String toString(){
```

Comment:
nit
can you add override annotation here

Reply:
right, adding it!